modules/net-vpc-firewall-yaml/README.md

```diff
@@ -4,47 +4,56 @@ This module allows creation and management of different types of firewall rules
 
 Yaml abstraction for FW rules can simplify users onboarding and also makes rules definition simpler and clearer comparing to HCL.
 
-Nested folder structure for yaml configurations is supported, which allows better and structured code management. 
+Nested folder structure for yaml configurations is supported, which allows better and structured code management for multiple teams and environments. 
```

Comment:
This is nice, but I wonder if the `common` folder is taking the place of hierarchical firewall rules. Would it make sense to add one sentence explaining the rationale of using VPC rules for common instead of hierarchical, maybe citing some limitations of hierarchical firewalls?

Reply:
Good point, done

Reply:
It's done in the example, not in the module itself